TheJumpCloud/jumpcloud-ADMU
# Sa 3209 set last login user

## Issues
* [SA-3209](https://jumpcloud.atlassian.net/browse/SA-3209) - SA-3209-Set-LastLoggedOnUser

## What does this solve?
1. Adds functionality to set the last logged on user to the migrated user. Default last logged on user is the migrated user.
2. Replace exe installer to msi installer 
## Is there anything particularly tricky?
n/a
## How should this be tested?
1. Run ADMU in a machine without JC agent:
* Select Install agent
* Migrate ADMU user to JC
2. Validate
* Agent installed
* Last logged on user registry should have migrated user SID and username (HKLM:\Software\Microsoft\Windows\CurrentVersion\Authentication\LogonUI)


[SA-3209]: https://jumpcloud.atlassian.net/browse/SA-3209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
